src/Integration.UnitTests/Binding/BindingProcessImplTests.cs

```diff
@@ -58,7 +56,7 @@ public class BindingProcessImplTests
         private ConfigurableSolutionRuleSetsInformationProvider ruleSetsInformationProvider;
         private ConfigurableHost host;
         private Mock<IFolderWorkspaceService> folderWorkspaceService;
-        private Mock<IJsTsProjectTypeIndicator> jstsIndicator;
```

Comment:
Getting rid of `IJsTsProjectTypeIndicator` - nice.

That should have been an indicator to whoever originally wrote the tests that something was wrong, since `IJsTsProjectTypeIndicator` is not a dependency of the type being tested here, `BindingProcessImpl`.